src/main/java/msg/practica/ro/model/Apartment.java

```diff
@@ -44,7 +44,7 @@
     private String description;
 
     @JoinColumn(name = "OWNER_ID")
-    @ManyToOne(cascade = {CascadeType.ALL})
+    @ManyToOne(cascade = {CascadeType.MERGE})
```

Comment:
Maybe this could be mapped as a list with @OneToMany (in the Owner Entity as a List of Apartments).
Please check the front-end requirements.